src/main/resources/assets/tconstruct/book/mighty_smelting/en_us/abilities/interact/bucketing.json

```diff
@@ -9,21 +9,21 @@
     },
 	{
       "text": "§lDear God.§r",
-	  "pargraph": true
+	  "paragraph": true
```

Comment:
Going to need a screenshot of this one to make sure it still looks right, the fact there was a typo in the JSON key implies we were relying on paragraph being false, so it might have too much spacing.

Reply:
Just tested myself as I am looking to do a 1.16.5 release soon, the typo was being relied on in code, so removing paragraph is the proper thing here